test/mock/ccd/services/otherEventStartEvent.js

```diff
@@ -1,13 +1,24 @@
+function getCurrentState(params) {
+  switch (params.eventType) {
+    case 'editAppeal':
+      return 'appealStarted';
+    case 'submitAppeal':
+      return 'appealStarted';
```

Comment:
Should this be return 'appealSubmitted'

Reply:
For each event we make a start event call and a submit event call. The start event gives us a token and the submit actually causes the event to happen. This is the start so you stay in the current state you are in only on the submit do you move.